documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -274,22 +277,22 @@ next_entry:
       jnc read_entry		# Till done
 ....
 
-.[.filename]#sys/boot/i386/boot0/boot0.S# [[boot-boot0-partition-scan]]
+.[.filename]#stand/i386/boot0/boot0.S# [[boot-boot0-partition-scan]]
 It is important to note that the active flag for each entry is cleared, so after the scanning, _no_ partition entry is active in our memory copy of [.filename]#boot0#. Later, the active flag will be set for the selected partition. This ensures that only one active partition exists if the user chooses to write the changes back to disk.
 
 The next block tests for other drives. At startup, the BIOS writes the number of drives present in the computer to address `0x475`. If there are any other drives present, [.filename]#boot0# prints the current drive to screen. The user may command [.filename]#boot0# to scan partitions on another drive later.
 
 [.programlisting]
 ....
       popw %ax			# Drive number
-      subb $0x79,%al		# Does next
-      cmpb 0x475,%al		#  drive exist? (from BIOS?)
+      subb $0x80-0x1,%al	# Does next
+      cmpb NHRDRV,%al		#  drive exist? (from BIOS?)
```

Comment:
There's a tab that gets removed here, which I don't think is what's intended as it moves the comment.

Reply:
I'll fix it.

Reply:
I fixed this.